### PR TITLE
add To-Do App

### PR DESCRIPTION
**Pull request by zuna2005**
Created a simple to-do app with the following functionality:
- add a task
- edit a task
- mark a task as done
- delete a task

Used bootstrap for the styles.

### CODE REVIEW

**Comment by shokhrukh-dsr** [comment]
I've changed my mind: you should always provide a description inside the PR. You can include a short overview of your changes. And some heads-ups or edge-cases or something like if there is any.

File: dom-traversal/domTraversal.html
Comment (shokhrukh-dsr) [inline dom-traversal/domTraversal.html:1]: What is this file doing here in this PR? `main` branch already has it.

File: domTraversal.html
Comment (shokhrukh-dsr) [inline domTraversal.html:1]: What's this file?

Comment (zuna2005) [reply domTraversal.html:1]: I'm a little confused, somehow my old commits were cloned to the new branch when I created it, even though I'm not sure what exactly I did wrong.

Comment (shokhrukh-dsr) [reply domTraversal.html:1]: Ok, don't do anything about it for now, let's discuss it tomorrow

**Pull request closed without merging**